src/camera.js

```diff
@@ -274,11 +274,11 @@ class IsometricCamera extends Camera {
     }
 
     setupProgram(program) {
-        program.uniform('Matrix4fv', 'u_projection', false, this.projection_matrix);
+        program.uniform('Matrix4fv', 'u_projection', this.projection_matrix);
 
-        program.uniform('3f', 'u_eye', 0, 0, this.viewport_height);
+        program.uniform('3fv', 'u_eye', [0, 0, this.viewport_height]);
```

Comment:
I don't think so, just confirming for my own understanding, since these
traditionally have different signatures but we've collapsed them (which is
fine).

I suppose it might suggest using the underlying GLSL types instead (float,
vec3, mat4, etc), but something to think about later.


On Thursday, April 21, 2016, David Valdman <notifications@github.com> wrote:

> In src/camera.js
> <https://github.com/tangrams/tangram/pull/298#discussion_r60670152>:
>
> >
> > -        program.uniform('3f', 'u_eye', 0, 0, this.viewport_height);
> > +        program.uniform('3fv', 'u_eye', [0, 0, this.viewport_height]);
>
> That's correct. Is there any advantage to one way or another?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram/pull/298/files/16531a3bd474e5155335cdaac5a8f180ccd551b9#r60670152>
>